qiskit_experiments/calibration/experiments/rabi.py

```diff
@@ -128,6 +139,27 @@ def _setup_fitting(self, **options) -> Union[Dict[str, Any], List[Dict[str, Any]
 
         return fit_options
 
+    def _run_analysis(
```

Comment:
Seems like we need to add this wrapper for many experiments. Though I think this should be a base class level change, current code assumes a very specific situation (i.e. calling signal processor). Can you think of how we can extend this to other experiments, such as running RB with custom processor? (this is not possible now because discriminator is lacking anyways).

Reply:
I think if 
- an experiment data entry has meas_level and meas_return
- all processors defines the behavior from the level0 (1 for now) data
- a processor can decide starting node based on above entry attributes

we can remove the processor getter function (get_to_signal_processor) and this method wrapper is no longer necessary.

Reply:
See here: 62256db